spec/controllers/sessions_controller_spec.rb

```diff
@@ -4,11 +4,23 @@
   it { should be_a Clearance::BaseController }
 
   describe 'on GET to /sessions/new' do
-    before { get :new }
+    context 'when a user is not signed in' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.